mscore/pianoroll/pianolevelschooser.ui

```diff
@@ -31,7 +37,13 @@
    <item>
     <widget class="QGroupBox" name="groupBox">
      <layout class="QGridLayout" name="gridLayout">
-      <item row="2" column="1">
+       <property name="leftMargin">
+         <number>5</number>
+       </property>
+       <property name="rightMargin">
+         <number>5</number>
+       </property>
+       <item row="2" column="1">
```

Comment:
Now looks like this:

![image](https://user-images.githubusercontent.com/5364225/134428780-38e68c7a-8caf-4906-986b-b4b3703440de.png)

(Previously, neither the combobox or spinbox were wide enough)